guides/source/plugins.md

```diff
@@ -458,13 +449,13 @@ Gem plugins currently in development can easily be shared from any Git repositor
 commit the code to a Git repository (like GitHub) and add a line to the Gemfile of the application in question:
 
 ```ruby
-gem 'yaffle', git: 'git://github.com/yaffle_watcher/yaffle.git'
+gem "yaffle", git: "https://github.com/rails/yaffle.git"
```

Comment:
I've changed it guided by that one day there might be a user by https://github.com/yaffle_watcher